src/components/Home/feed.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import NewTweetForm from "./newTweetForm";
+
+const renderTweeet = (currentTweet) => {
+  return (
+      <div className="box">
+        <div className="panel-block has-background-grey-lighter">
+          <div className="media-left">
+            <img style={{borderRadius: '50%'}} className="image is-32x32" src={currentTweet.user.image} alt=""/>
+          </div>
+          <div className="media-content">
+            <b>{currentTweet.user.name}</b><br/>
+            <small>{currentTweet.user.username}</small>
+          </div>
+        </div>
+        <article className="media">
+          <div className="content" style={{width: '100%'}}>
+            <blockquote>{currentTweet.tweet}</blockquote>
+          </div>
+        </article>
+      </div>
+  );
+};
+
+const Feed = (props) => {
```

Comment:
`Feed` is singular. Why is it rendering all the tweets?

Reply:
In this case move the `renderTweet`  body to `render` function only. It is just a redirection and not helpful to have it as a function.